edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -429,14 +434,15 @@ internal static bool CheckIfTwinSignatureIsValid(TwinCollection twinDesiredPrope
             return true;
         }
 
-        internal static bool CheckIfTwinPropertiesAreSigned(TwinCollection twinDesiredProperties)
+        internal bool CheckIfManifestSigningIsEnabled(TwinCollection twinDesiredProperties)
         {
-            // If there is no integrity section in the desired twin properties then it is unsigned twin
+            // If there is no integrity section in the desired twin properties and the manifest trust bundle is not configured then manifest signing is turned off
+            // If we have integrity section or the configuration of manifest trust bundle then manifest signing is turned on
             JToken integrity = JObject.Parse(twinDesiredProperties.ToString())["integrity"];
-            return integrity != null && integrity.HasValues != false;
+            return this.manifestTrustBundle.HasValue != false || (integrity != null && integrity.HasValues != false);
```

Comment:
I prefer `this.manifestTrustBundle.HasValue || (integrity != null && integrity.HasValues)`

Reply:
Same comment on the rest of this file. I prefer `this.manifestTrustBundle.HasValue` and `!this.manifestTrustBundle.HasValue` over comparing to true and false.  It's more readable and understandable.